src/main/java/Constants.java

```diff
@@ -0,0 +1,20 @@
+public class Constants {
+    public static final String horizontalLine = "    ___________________________________________";
+    public static final String helloMessage = "Hello! I'm Duke";
+    public static final String assistMessage = "What can I do for you?";
+    public static final String byeMessage = "Bye. Hope to see you again soon!";
+    public static final String doneMessage = "Nice! I've marked this task as done:";
+    public static final String addedMessage = "Got it. I've added this task:";
+    public static final String errorMessage = "Invalid input!";
+    public static final String doneErrorMessage = "Done what? Input the task number please!";
+    public static final String todoErrorMessage = "What is the task? Please add one!";
+    public static final String deadlineErrorMessage = "When is the deadline? Please add one!";
+    public static final String eventErrorMessage = "When is the event? Please add one!";
```

Comment:
It may be better to use upper case for all constants and use underscore to seperate words